XXX revisit when our gtk4 supports dma-buf.
Fall back to gdk_gl_texture_new() when GDK lacks dmabuf support. GTK only
implements GdkDmabufTexture when built with <linux/dma-buf.h> (HAVE_DMABUF),
which is not the case on OpenBSD: gdk_dmabuf_texture_builder_build() always
fails and GDK advertises no dmabuf formats. Import the buffer as an EGLImage
ourselves instead.

Index: Source/WebKit/UIProcess/gtk/AcceleratedBackingStore.cpp
--- Source/WebKit/UIProcess/gtk/AcceleratedBackingStore.cpp.orig
+++ Source/WebKit/UIProcess/gtk/AcceleratedBackingStore.cpp
@@ -670,9 +670,11 @@ void AcceleratedBackingStore::didCreateDMABufBuffer(ui
 #endif
 
 #if GTK_CHECK_VERSION(4, 13, 4)
-    if (auto buffer = BufferDMABuf::create(*webPage, id, m_surfaceID, size, usage, format, WTF::move(fds), WTF::move(offsets), WTF::move(strides), modifier)) {
-        m_buffers.add(id, WTF::move(buffer));
-        return;
+    if (gdk_dmabuf_formats_get_n_formats(gdk_display_get_dmabuf_formats(gtk_widget_get_display(webPage->viewWidget())))) {
+        if (auto buffer = BufferDMABuf::create(*webPage, id, m_surfaceID, size, usage, format, WTF::move(fds), WTF::move(offsets), WTF::move(strides), modifier)) {
+            m_buffers.add(id, WTF::move(buffer));
+            return;
+        }
     }
 #endif
 
